README.md

```diff
@@ -14,6 +14,12 @@ build the first 2 sections of the mobile website using the template you chose in
 - git and gitHub
 - Visual Code
 
```

Comment:
### OPTIONAL

-  Well done making your readme file descriptive, kindly ensure all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) are present 👍🏿